confluence/types/bodies/bodies.go

```diff
@@ -0,0 +1,195 @@
+package bodies
```

Comment:
Why do we need an entirely new package for these rather than just a new file in the `confluence` package? Types packages irk me lol

Reply:
Is that possible?
I'll try it.

Reply:
completed